parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -73,6 +73,32 @@ NVIDIA_DOCKER_VERSION=2.0.3
 DOCKER_VERSION=1.13.1-1
 NVIDIA_CONTAINER_RUNTIME_VERSION=2.0.0
 
+configure_prerequisites() {
+    if which apt > /dev/null; then
+        # Install required packages; only for apt-based systems
+        apt update -qq
+        apt install -y curl gpg jq nfs-common
+    fi
+
+    modprobe br_netfilter overlay
+    check_and_fix_kernel_parameter /proc/sys/net/ipv4/ip_forward net.ipv4.ip_forward 1
+}
+
+check_and_fix_kernel_parameter() {
```

Comment:
I did a quick audit of the code to see if we can use this new `check_and_fix_kernel_parameter` func to generalize existing foo, didn't find any. If you could double-check that, 👍. (If you find any opportunities, feel free to refactor to use this new func.)

Reply:
Looks like the codebase already uses sysctl-d-60-CIS.conf for all the sysctl needs.I could move the setting there, but don't recall if it will get priority over

Reply:
So this is tricky... /etc/sysctl.d/99-sysctl.conf is symlinked to /etc/sysctl.conf where the ip_forward setting resides. So the settings in 60-CIS do not supersede it.
With this setting there by default (and thus present at boot time in /proc) there's no alternative but to remove it from /etc/sysctl.conf and update /proc. I welcome recommendations.

Reply:
The current fix works, so unless there are stylistic comments I'll consider this resolved.